confluent_kafka_helpers/tracing/datadog.py

```diff
@@ -0,0 +1,39 @@
+import os
```

Comment:
This is a bit unfortunate; optimally, the library shouldn't know about the SDK (Datadog in this case), but since Datadog doesn't fully map OTEL semantics we need to do something about it.